CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ ExternalProject_Add(opencv3_src
     -DBUILD_opencv_dnn=OFF # Pulls in the system protobuf as a dependency!
     -DBUILD_opencv_dnns_easily_fooled=OFF
     -DBUILD_opencv_cnn_3dobj=OFF
-    -DBUILD_opencv_aruco=OFF
+    -DBUILD_opencv_aruco=ON
```

Comment:
will this create a new library? if so, please add it also to the `cs_export` command in #17 after you merge this

Reply:
ok I will add it